src/dmd/target.h

```diff
@@ -89,12 +89,16 @@ struct Target
     FPTypeProperties<double> DoubleProperties;
     FPTypeProperties<real_t> RealProperties;
 
+private:
+    Type *va_list;
+
+public:
     void _init(const Param& params);
     // Type sizes and support.
     unsigned alignsize(Type *type);
     unsigned fieldalign(Type *type);
     unsigned critsecsize();
-    Type *va_listType();  // get type of va_list
+    Type *va_listType(const Loc &loc, Scope *sc);  // get type of va_list
```

Comment:
As expected, my code is now littered with `target.va_listType(Loc (), NULL)`.  _Sigh_.

Reply:
What's stopping you from adding a GDC-specific param-less overload? - LDC has one single call to this outside of the frontend...